widget_driver/example/lib/widgets/coffee_community/coffee_community_page_driver.dart

```diff
@@ -8,12 +8,12 @@ part 'coffee_community_page_driver.g.dart';
 
 @GenerateTestDriver()
 class CoffeeCommunityPageDriver extends WidgetDriver {
-  final AuthService _authService;
+  late AuthService _authService;
   StreamSubscription? _subscription;
 
-  CoffeeCommunityPageDriver(BuildContext context)
-      : _authService = context.watch<AuthService>(),
-        super(context) {
+  @override
+  void didUpdateBuildContext(BuildContext context) {
+    _authService = context.read<AuthService>();
```

Comment:
Don't you need to cancel a (possible) previous subscription here?
```suggestion
    _authService = context.read<AuthService>();
    await _subscription?.cancel();
    
```

Reply:
Well, since the `context.read<AuthService>();` is a read only.. not a subscription.. 
The `didUpdateBuildContext` will only be called once..
But we could change it to handle a potential second call..  We can have a chat maybe about it